index.js

```diff
@@ -1,13 +1,42 @@
 'use strict'
 
+const back = require('./lib/back')
+const emitter = require('./lib/global_emitter')
+const {
+  activate,
+  isActive,
+  isDone,
+  pendingMocks,
+  activeMocks,
+  removeInterceptor,
+  disableNetConnect,
+  enableNetConnect,
+  removeAll: cleanAll,
```

Comment:
Can we just rename `removeAll` to `cleanAll` and have it consistent everywhere?

Reply:
I was thinking about that. It would make the naming inside `intercept.js` a bit inconsistent so I thought it might be better to defer it for a future round of renames / refactoring. I don't feel strongly about that, though.

Reply:
To expand on that, I'll add that I'm not feeling awesome about the contents of `intercept.js`. It feels like a bit of a hodgepodge, with at least two concerns (albeit related).

I think part of it could go into something like a MockRequestStore which holds all the interceptors, though `overrideClientRequest` would still need a good home. Maybe it could be split into `mock_request_store.js` for the global interceptor store and `activation.js` for the activation bits? 🤔 

There are `remove` and ` removeInterceptor` functions in that file, though I don't mind changing `removeAll` to `cleanAll` if you think it's better.